diffpy/snmf/containers.py

```diff
@@ -36,13 +36,18 @@ def apply_stretch(self, m):
         Returns
         -------
         tuple of 1d arrays
-          The tuple of vectors where one vector is the stretched component, one vector is the 1st derivative of the
-          stretching operation, and one vector is the second derivative of the stretching operation.
+          The tuple of vectors where one vector is the stretched component, one vector is the 1st derivative
+          of the stretching operation, and one vector is the second derivative of the stretching operation.
         """
         normalized_grid = np.arange(len(self.grid))
-        func = lambda stretching_factor: np.interp(
-            normalized_grid / stretching_factor, normalized_grid, self.iq, left=0, right=0
-        )
+        # func = lambda stretching_factor: np.interp(
+        #     normalized_grid / stretching_factor, normalized_grid, self.iq, left=0, right=0
+        # )
+
+        # E731 do not assign a lambda expression, use a def
```

Comment:
remove this comment.  Less chatter is better in the future, though I appreciate you wanting to communicate with me what you are doing (I think I can figure it out though.

For these lambda flake8 errors we would normally see if the function is being used in more than one place, and if it is, define a private function somewhere outside the main part of the code.  If it is only being used in one place, we would simply move the lambda to where it is being used (so not assign it to a variable and then put the variable where it is being used, or we could put a `# noqa: E731` at the end of this line so we leave it as is but it won't fail flake8. 

How do we decide?  the objective of all of this is not to pass flake8 but to make the code better, so the choice we make is the one that will make the code the best.

Also, in general, don't leave commented code in there, just delete it.  Again, I appreciate the impulse (more transparency) but we don't want to carry around every old version of code in hte future.

Reply:
Thank you. A new pull request addressing your feedback has been made: https://github.com/diffpy/diffpy.snmf/pull/56